### PR TITLE
Importer data copy

### PR DESCRIPTION
**Pull request by geofjamg**

### CODE REVIEW

File: iidm-converter-api/src/main/java/eu/itesla_project/iidm/parameters/ParameterDefaultValueConfig.java [hunk 1]
```diff
@@ -23,28 +23,48 @@
...
+    }
+
+    private synchronized ModuleConfig getModuleConfig() {
+        if (!init) {
+            if (platformConfig == null) {
+                platformConfig = PlatformConfig.defaultConfig();
+            }
+            configCache = platformConfig.moduleExists(MODULE_NAME) ? platformConfig.getModuleConfig(MODULE_NAME) : null;
+        }
+        init = true;
```
Comment (mathbagu) [inline iidm-converter-api/src/main/java/eu/itesla_project/iidm/parameters/ParameterDefaultValueConfig.java]: This line should be inside the if block.

File: iidm-xml-converter/src/test/java/eu/itesla_project/iidm/xml/XMLImporterTest.java [hunk 2]
```diff
@@ -0,0 +1,154 @@
...
+
+    @Test
+    public void copy() throws Exception {
+        importer.copy(new FileDataSource(fileSystem.getPath("/"), "test0"), new FileDataSource(fileSystem.getPath("/"), "test0_copy"));
+        assertTrue(Files.exists(fileSystem.getPath("/test0_copy.xiidm")));
+
+        // test copy with id mapping file
+        importer.copy(new FileDataSource(fileSystem.getPath("/"), "test6"), new FileDataSource(fileSystem.getPath("/"), "test6_copy"));
+        assertTrue(Files.exists(fileSystem.getPath("/test6_copy.xiidm")));
+        assertTrue(Files.exists(fileSystem.getPath("/test6_copy_mapping.csv")));
```
Comment (mathbagu) [inline iidm-xml-converter/src/test/java/eu/itesla_project/iidm/xml/XMLImporterTest.java:125]: Add a content comparison assertion?

File: iidm-converter-api/src/main/java/eu/itesla_project/iidm/parameters/ParameterDefaultValueConfig.java (hunk 1 above)
Comment (geofjamg) [reply iidm-converter-api/src/main/java/eu/itesla_project/iidm/parameters/ParameterDefaultValueConfig.java]: Done

File: iidm-xml-converter/src/test/java/eu/itesla_project/iidm/xml/XMLImporterTest.java (hunk 2 above)
Comment (geofjamg) [reply iidm-xml-converter/src/test/java/eu/itesla_project/iidm/xml/XMLImporterTest.java:125]: Done

**Pull request merged**